src/Germs.ts

```diff
@@ -0,0 +1,228 @@
+export { Germ, GermBlue, GermGreen, GermOrange, GermPink };
+
+import { constants, center } from './constants';
+import { StageVars } from './StageVars';
+import { germPoolMap, gameOver } from './scenes/Main';
+
+abstract class Germ extends Phaser.Physics.Arcade.Sprite {
+    unfreezeTimer: Phaser.Time.TimerEvent | null = null;
+    readyToReproduce = false;
+    frozen = false;
+
+    constructor(scene: Phaser.Scene, x: number, y: number, texture: string) {
+        super(scene, x, y, texture);
+        this.unfreeze();
+    }
+
+    abstract color(): string
+
+    die() {
+        this.disableBody(true, true);
+        this.setActive(false);
+
+        // Also clean up the object in preparation for its reuse from the pool.
+        this.readyToReproduce = false;
+        this.unfreeze();
+    }
+
+    freeze() {
+        if (this.unfreezeTimer) this.unfreezeTimer.destroy();
```

Comment:
brackets or bust baby

Reply:
actually i think you could do this.unfreezeTimer?.destroy() or something like that.